src/app/services/sms/sms.service.ts

```diff
@@ -145,153 +160,187 @@ const getTwilio = async (
  * @param recipient The mobile number of the recipient
  * @param message The message to send
  */
-const send = async (
+const send = (
```

Comment:
should we rename this to something like `sendSms`? idk `send` is a little generic haha